### ISSUE TITLE
OpenID Authentication not working for YaHoo!

### ISSUE THREAD

**Issue by adebisi-fa** [Kinto/kinto#1716]
Hello,  

Thanks for this great framework!  It's pretty awesome!!!!

Please, kindly advise what else I need to do to activate a YaHoo! OpenID Connect Authentication integration, as all effort to configure this in my kinto.ini hasn't been working.

Here is an excerpt of my configuration file.

```
multiauth.policies = account google yahoo
. . .
multiauth.policy.yahoo.use = kinto.plugins.openid.OpenIDConnectPolicy
multiauth.policy.yahoo.userid_field = email
multiauth.policy.yahoo.issuer = https://api.login.yahoo.com/oauth2/request_auth/
multiauth.policy.yahoo.client_id = <id goes here>
multiauth.policy.yahoo.client_secret = <secret goes here>
```

PLEASE NOTE:  Just to add that **Account** and **Google** (OpenID Connect) has been pretty functional. 

Thanks and regards, please.

**Comment by adebisi-fa** [comment]
Woops! **Kinto** all the way.  It worked!

Thanks @Natim! So so much grateful!

Thanks, please.

**Comment by Natim** [comment]
You are welcome, thank you for using Kinto <3

**Comment by adebisi-fa** [comment]
Would be nice to have all acceptable issuer values for the common OpenID Connect Providers, at least, in the documentation.

Spent a good part of today looking around for this value.  And the providers documentations didn't help either.

Thanks, once again @Natim.

**Comment by Natim** [comment]
@adebisi-fa Can I let you start a pull-request to add the one you successfully tried?

**Comment by adebisi-fa** [comment]
I'll do that, asap.  Thanks!

As a side note, I noticed that the Yahoo OpenID Provider doesn't support the `email` scope, as such  I couldn't login into the kinto-admin with it. 

A careful examination also shows that specifying only `openid` is enough to get the `email` claim from the provided Yahoo JWT token. (This is unlike some providers, `Google`, that requires explicit `email` scope to get the email into their token).

To solve login into `kinto-admin`, can OpenID providers scopes for the `kinto-admin` plugin be configurable, instead of using the generic `openid`, and `email`, for enabled providers?

[I added this here to keep all Yahoo OpenID issues in one place.]
